syft/pkg/cataloger/java/tar_wrapped_archive_parser_test.go

```diff
@@ -0,0 +1,53 @@
+package java
+
+import (
+	"os"
+	"path"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/stretchr/testify/assert"
```

Comment:
```suggestion
	"github.com/stretchr/testify/assert"
	"github.com/stretchr/testify/require"
```

Reply:
I'll incorporate, but not through the GitHub UI --it doesn't DCO suggested commits which is annoying :(